lib/dolly/request.rb

```diff
@@ -88,6 +92,10 @@ def values_to_json hash
     def full_path resource
       "/#{database_name}/#{resource}"
     end
+
+    def attachment_path resource, attachment_name
+      "#{full_path(resource)}/#{attachment_name}"
```

Comment:
@neojin I think I'm going to do that in a refactor PR since the id_as_resource is currently escaped here https://github.com/amco/dolly/blob/1.0/lib/dolly/document.rb#L98, and I wanted to stay consistent with it.  Same goes fore the `full_path` -> `resource_path`